src/Kafka/ManageKafkaPermissions/components/ResourceTypeLabel.css

```diff
@@ -1,7 +1,7 @@
 .pf-c-label.mas-m-solid {
-  --pf-c-label__content--before--BorderColor: transparent;
+  --pf-c-label__content--before--BorderColor: transparent !important;
```

Comment:
@jgiardino One way of fixing the "mas-m-solid" issue I found was adding "!important" to the necessary lines in the original class in "ResourceTypeLabel.css", but not sure if this okay way to solve it.